mappings/net/minecraft/entity/mob/Hoglin.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5137 net/minecraft/entity/mob/Hoglin
```

Comment:
What is this? `ZoglinEntity` or `HoglinLogic`?

Reply:
It's an interface that both `HoglinEntity` and `ZoglinEntity` implement. Couldn't think of a good name lol